README.md

```diff
@@ -1,58 +1,274 @@
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/lix-pm/Lobby)
 
-# lix - the haxe package manager that rox ... ba-dum-tss ...
+# Lix - a dependable package manager for your Haxe projects
 
-In a nutshell, lix is an attempt to get dependency management right, building on lessons learnt from looking at NPM, Cargo and failure to move haxelib forward. The core proposition of lix is that **dependencies should be fully locked down and versioned, so that every state can be reliably replicated.**
+Lix is a package manager that makes it easy for you to track dependencies used in your Haxe project, and to make sure everyone always has the right versions when they build your app.
+
+- Lix installs dependencies faster than haxelib.
```

Comment:
Does this need further clarification?  ("... because it uses a global cache and streaming decompression, you'll notice things are way faster")